File: print/hplip/files/patch-base_utils.py
--- base/utils.py.orig	2022-02-23 07:41:04 UTC
+++ base/utils.py
@@ -139,11 +139,11 @@ for s in EXPECT_WORD_LIST:
 def get_cups_systemgroup_list():
     lis = []
     try:
-        fp=open('/etc/cups/cupsd.conf')
+        fp=open('/usr/local/etc/cups/cupsd.conf')
     except IOError:
         try:
-            if "root" != grp.getgrgid(os.stat('/etc/cups/cupsd.conf').st_gid).gr_name:
-                return [grp.getgrgid(os.stat('/etc/cups/cupsd.conf').st_gid).gr_name]
+            if "root" != grp.getgrgid(os.stat('/usr/local/etc/cups/cupsd.conf').st_gid).gr_name:
+                return [grp.getgrgid(os.stat('/usr/local/etc/cups/cupsd.conf').st_gid).gr_name]
         except OSError:
             return lis
 

File: news/newspost/files/patch-base_utils.c
--- base/utils.c.orig	2003-04-23 15:33:23 UTC
+++ base/utils.c
@@ -43,7 +43,7 @@ file_entry * file_entry_free(file_entry 
 	return NULL;
 }
 
-Buff * getline(Buff *buff, FILE *file){
+Buff * get_line(Buff *buff, FILE *file){
 	char c = fgetc(file);
 	buff = buff_free(buff);
 	while(TRUE){
